listenbrainz/feedback.py

```diff
@@ -0,0 +1,24 @@
+import uuid
+
+from typing import List
+from pydantic import BaseModel, ValidationError, validator
+
+
+class Feedback(BaseModel):
+    user_id: int
+    recording_msid: str
+    score: int
+
+    @validator('score')
+    def check_score_is_valid(cls, scr):
+        if scr not in [-1, 0, 1]:
+            raise ValueError('Score can have a value of 1, 0 or -1.')
+        return scr
+
+    @validator('recording_msid')
+    def check_recording_msid_is_valid_uuid(cls, rec_msid):
+        try:
+            rec_msid = uuid.UUID(rec_msid)
+            return rec_msid
```

Comment:
just a thing to be aware of here, this will be a UUID object, not a string. In most cases this probably won't be a problem - postgres will happily accept it as a value, but it does mean that the type definition is incorrect. Also, you might find that some places don't like it.
Maybe better to use `str(rec_msid)` here

Reply:
Makes sense